hadoop-ozone/dist/src/shell/ozone/ozone

```diff
@@ -138,7 +125,7 @@ function ozonecmd_case
     ;;
     freon)
       OZONE_CLASSNAME=org.apache.hadoop.ozone.freon.Freon
-      OZONE_FREON_OPTS="${OZONE_FREON_OPTS} ${NETTY_OPTS} ${OZONE_MODULE_ACCESS_ARGS}"
+      OZONE_FREON_OPTS="${OZONE_FREON_OPTS} ${OZONE_MODULE_ACCESS_ARGS}"
```

Comment:
Should we add `${RATIS_OPTS}` here and similar commands, where `NETTY_OPTS` was present, but `RATIS_OPTS` is not?  Otherwise `tryReflectionSetAccessible=true` will not be set for such commands.

Reply:
Yes, we have to add `${RATIS_OPTS}` in place of `$NETTY_OPTS`.